src/pages/components/NewIdeaCard/compound/Profile/Profile.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled';
+import {
+  Box,
+  SVGLoginDefaultProfile,
+  SVGScrap24,
+  SVGScrapFilled24,
+  Spacer,
+  Text,
+  theme,
+} from 'concept-be-design-system';
+import { MouseEventHandler, useState } from 'react';
+
+import { useDeleteBookmarkIdea } from '../../../../Feed/hooks/mutations/useDeleteBookmarkIdea';
+import { usePostBookmarkIdea } from '../../../../Feed/hooks/mutations/usePostBookmarkIdea';
+import { formatCommentDate } from '../../../../Feed/utils/formatCommentDate';
+import { useIdeaId, useProfileContext } from '../../NewIdeaCardContext';
+
+const Profile = () => {
+  const id = useIdeaId();
+  const { profileImageUrl, nickname, skills, isBookmarked, createdAt } = useProfileContext();
+  const { postBookmarkIdea } = usePostBookmarkIdea();
+  const { deleteBookmarkIdea } = useDeleteBookmarkIdea();
+
+  // 이미지 로딩 실패를 처리하기 위한 상태
+  const [imageError, setImageError] = useState(false);
+
+  // 이미지 로딩 실패 처리 함수
+  const handleImageError = () => {
+    setImageError(true); // 이미지 로딩 실패 상태를 true로 설정
+  };
+
+  const bookmarkIdea: MouseEventHandler<SVGSVGElement> = (e) => {
+    e.stopPropagation();
+    postBookmarkIdea(id);
+  };
+
+  const unbookmarkIdea: MouseEventHandler<SVGSVGElement> = (e) => {
+    e.stopPropagation();
+    deleteBookmarkIdea(id);
+  };
+
+  return (
+    <ProfileWrapper>
+      <ProfileBox>
+        {imageError === true ? (
+          <SVGLoginDefaultProfile />
+        ) : (
+          <Box width={36} height={36} overflow="hidden" borderRadius="0 150px 150px 0">
+            <Img src={profileImageUrl} onError={handleImageError} />
+          </Box>
```

Comment:
svg 파일을 디자인시스템에서 React Component로 내보내서 상태로 분기처리가 필요하군요. default로 string으로 뺀다음 불러오는 쪽에서 React Component로 활용하는 방법을 사용하면 다음과 같은 구문으로 처리해볼 수도 있겠네요. svg 파일을 React Component로 export 한거에 불편함은 없으신가요?

```suggestion
          <Box width={36} height={36} overflow="hidden" borderRadius="0 150px 150px 0">
            <Img
                src={profileImageUrl}
                onError={(e: SyntheticEvent<HTMLImageElement, Event>) => {
                  e.currentTarget.src = SVGLoginDefaultProfile;
            }} />
          </Box>
```

Reply:
넵 불편함은 없는데, 이 방법 괜찮은 것 같습니다.
적용해볼게요.

Reply:
넵 아마 타입쪽에서 에러도 발생할 것 같네요. SVG는 ReactComponent로 인식되어서 as string 을 하지 않는 이상..